### PR TITLE
Fix external_db.txt

### PR DESCRIPTION
**Pull request by mira13**
## Description

Some test dbs has wrong newlines. They are removed in this PR

## Use case

travis build, unit tests are not running correctly

## Benefits

Correct unit testing

## Possible Drawbacks

not found

## Testing

no

_If so, do the tests pass/fail?_

_Have you run the entire test suite and no regression was detected?_

yes

### CODE REVIEW

File: modules/t/test-genome-DBs/parus_major1/core/external_db.txt [hunk 1]
```diff
@@ -428,11 +424,11 @@ The main requirement behind this entry, is to be able to link TAIR GO annotation
...
 50847	Ngyuen_et_al_2012	\N	PRED	0	PMID 23153364	MISC	\N	\N	The complete swine olfactory subgenome: expansion of the olfactory gene repertoire in the pig genome. BMC Genomics201213:584 PMID: 23153364
 50848	Plant_Reactome_Reaction	\N	KNOWNXREF	0	Plant Reactome Reaction ID	MISC	\N	\N	Plant Reactome is a database of plant metabolic and regulatory pathways maintained by Gramene. This ID links to a Reaction in Plant Reactome.
 50849	Plant_Reactome_Pathway	\N	KNOWNXREF	0	Plant Reactome Pathway ID	MISC	\N	\N	Plant Reactome is a database of plant metabolic and regulatory pathways maintained by Gramene. This ID links to a Pathway in Plant Reactome.
-50850	CGSNL	\N	KNOWNXREF	0	Committee on Gene Symbolization, Nomenclature and Linkage of the Rice Genetics Cooperative	MISC	\N	\N	The Committee on Gene Symbolization, Nomenclature and Linkage (CGSNL) of the Rice Genetics Cooperative has revised the gene nomenclature system for rice (Oryza) to take advantage of the completion of the rice genome sequence and the emergence of new methods for detecting, characterizing, and describing genes in the biological community (<A HREF="https://doi.org/10.1007/s12284-008-9004-9">McCouch 2008</A>).
```
Comment (nwillhoft) [inline modules/t/test-genome-DBs/parus_major1/core/external_db.txt:435]: The proposed change applied to this DOI breaks the link. Please can the DOI link be kept as it currently is, i.e. "https://doi.org/10.1007/s12284-008-9004-9"

**Review by nwillhoft — CHANGES_REQUESTED** [review]
Have made an in-line comment about a DOI link needing to stay as it currently is (because a '/' replaced with a space is breaking the link). All other changes look good. Build checks are passing.

**Review by nwillhoft — APPROVED** [review]
The removal of extra new line and white space characters looks good.

**Review by TamaraNaboulsi — APPROVED** [review]
All wrong newlines removed from files, looks good.

**Pull request merged**